src/org/apache/storm/metrics2/Aggregation.java

```diff
@@ -56,6 +56,15 @@ public void filterComp(String comp) {
         this.settings.put("compId", comp);
     }
 
+    public void filterTimeStart(String time) { this.settings.put("startTime", time);}
```

Comment:
I meant that the interface here should also be longs. That way the user of the class is forced to provide a number..